Realm/Realm/Configurations/RealmConfiguration.cs

```diff
@@ -49,17 +49,6 @@ public class RealmConfiguration : RealmConfigurationBase
         /// </param>
         public delegate void MigrationCallbackDelegate(Migration migration, ulong oldSchemaVersion);
 
-        /// <summary>
-        /// A callback, invoked when opening a Realm for the first time during the life
-        /// of a process to determine if it should be compacted before being returned
-        /// to the user.
-        /// </summary>
-        /// <param name="totalBytes">Total file size (data + free space).</param>
-        /// <param name="bytesUsed">Total data size.</param>
-        /// <returns><c>true</c> to indicate that an attempt to compact the file should be made.</returns>
-        /// <remarks>The compaction will be skipped if another process is accessing it.</remarks>
-        public delegate bool ShouldCompactDelegate(ulong totalBytes, ulong bytesUsed);
```

Comment:
these are moved to RealmConfigurationBase so they become available to sync configs.